src/resources/postcss/tribe-common-admin/_main.pcss

```diff
@@ -189,6 +189,8 @@ table.plugins {
 
 .tribe-settings-form {
 	max-width: 1000px;
+	border-inline: 1px solid var(--tec-admin-core-grey-10);
+	border-bottom: 1px solid var(--tec-admin-core-grey-10);
```

Comment:
Why no border top?
```suggestion
	border: 1px solid var(--tec-admin-core-grey-10);
```

Reply:
Then you dan't have to mess with borders on the `<li>`

Reply:
> Then you don't have to mess with borders on the `<li>`

The `<li>` is positioned on top of this element with `position: absolute`. This seems to cover the upper borders of the `.tribe-settings-form` element.

With some testing, it looks like we need the full border on `.tribe-settings-form`, and also the borders on the `<li>`. Here's a screenshot with *only* the form borders:

<img width="961" alt="Screenshot 2024-08-21 at 21 19 14" src="https://github.com/user-attachments/assets/4b697558-4fd5-4761-a126-b18329633912">

Here's a screenshot with the `<li>` borders added:

<img width="961" alt="Screenshot 2024-08-21 at 21 20 06" src="https://github.com/user-attachments/assets/2b86889f-beb1-46e2-b19a-89dd7d0c04c5">